cli-output/src/cli_output.rs

```diff
@@ -1625,6 +1628,19 @@ impl fmt::Display for CliSignOnlyData {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         writeln!(f)?;
         writeln_name_value(f, "Blockhash:", &self.blockhash)?;
+        if let Some(transaction) = self.transaction.as_ref() {
+            let signature_completeness = if self.absent.is_empty() && self.bad_sig.is_empty() {
```

Comment:
both conditions need to be on `self.bad_sig.is_empty()` right?  what good is dumping the serialization if one of the sigs is bad?